docs/src/main/tut/middleware.md

```diff
@@ -4,35 +4,34 @@ title: Middleware
 weight: 115
 ---
 
-A middleware is a wrapper around a [service] that provides a means of manipulating
-the `Request` sent to service, and/or the `Response` returned by the service. In
-some cases, such as [Authentication], middleware may even prevent the service
-from being called.
+A `Middleware` wraps a [`Service`]. Doing so allows you to change the `Request`
+that is sent to a `Service`, and/or modify the `Response` that is returned by
+that `Service`. In some cases, such as [Authentication], `Middleware` might
+prevent the `Service` from being called.
 
-At its most basic, middleware is simply a function that takes one `Service` as a
-parameter and returns another `Service`. In addition to the `Service`, the middleware
-function can take any additional parameters it needs to perform its task. Let's look
-at a simple example.
+`Middleware` is a function that takes one `Service` and returns a different
+`Service`.
 
-For this, we'll need a dependency on the http4s [dsl].
+1. Add a dependency to SBT on the [dsl] by copying and pasting the following
+   code to the existing `build.sbt` file:
 
 ```scala
 libraryDependencies ++= Seq(
   "org.http4s" %% "http4s-dsl" % http4sVersion
 )
 ```
-and some imports.
+1. Add the following import statements:
 
 ```tut:silent
-import cats.effect._
+import cats.effect._ // To import the IO type, see cats-effect documentation.
```

Comment:
What does this mean?  To specialize on the IO type?  We've already imported the IO type here.

Reply:
"This import statement is to import the `IO` type. For more information about `IO`, see cats-effect."

Reply:
Why not use the specific import? I guess this is a bit redundant :)